pkgs/by-name/cl/cl-parse/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+}:
+
+buildGoModule rec {
+  pname = "cl-parse";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "scottmckendry";
+    repo = "cl-parse";
+    rev = "v${version}";
+    hash = "sha256-SZcr8fBTlA0Ibmx/8vVf1jmNkRTPC5nRtM2mOoFF/rg=";
+  };
+
+  vendorHash = "sha256-/SL6FE1rX1xkJ6vpVJUms7HUnLNY6qq66jaYUbGWKsM=";
+  nativeBuildInputs = [ installShellFiles ];
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  postInstall = ''
+    installShellCompletion --cmd cl-parse \
+      --bash <($out/bin/cl-parse completion bash) \
+      --zsh <($out/bin/cl-parse completion zsh) \
+      --fish <($out/bin/cl-parse completion fish)
+  '';
```

This fixes cross build:
```suggestion
  postInstall =
    let
      cl-parse =
        if stdenv.buildPlatform.canExecute stdenv.hostPlatform then
          placeholder "out"
        else
          buildPackages.cl-parse;
    in
    ''
      installShellCompletion --cmd cl-parse \
        --bash <(${cl-parse}/bin/cl-parse completion bash) \
        --fish <(${cl-parse}/bin/cl-parse completion fish) \
        --zsh <(${cl-parse}/bin/cl-parse completion zsh)
    '';
```

---

Thank you. I've applied the update.